spec/models/blog_spec.rb

```diff
@@ -4,51 +4,54 @@
 
 RSpec.describe Blog, type: :model do
 
-  let(:blog) { FactoryBot.build(:blog) }
-
-  context "Validations" do
-    context "check for presence" do
-      required_fields = ["title", "base_url"]
-      required_fields.each do |field|
-        it "raises an error when #{field} is not present" do
-          blog.update(field => "")
-          expect { blog.save! }.to raise_error(/#{field.humanize(capitalize: true)} can't be blank/)
+  describe "A basic blog" do
+    let(:blog) { FactoryBot.build(:blog) }
+
+    context "Validations" do
+      context "check for presence" do
+        required_fields = ["title", "base_url"]
+        required_fields.each do |field|
+          it "raises an error when #{field} is not present" do
+            blog.update(field => "")
+            expect { blog.save! }.to raise_error(/#{field.humanize(capitalize: true)} can't be blank/)
+          end
         end
       end
-    end
-    context "checks for a well formed url" do
-      it "raises and error when an invalid url is submitted" do
-        expect { FactoryBot.create(:blog_bad_url) }.to raise_error(/#{I18n.t('manifold.error.invalid_url')}/)
+      context "checks for a well formed url" do
+        it "raises and error when an invalid url is submitted" do
+          expect { FactoryBot.create(:blog_bad_url) }.to raise_error(/#{I18n.t('manifold.error.invalid_url')}/)
+        end
       end
     end
-  end
 
-  context "Default values" do
-    context "when no value is supplied for" do
-      context ":public_status" do
-        it "returns false" do
-          expect(blog.public_status).to be false
+    context "Default values" do
+      context "when no value is supplied for" do
+        context ":public_status" do
+          it "returns false" do
+            expect(blog.public_status).to be false
+          end
         end
-      end
-      context ":feed_path" do
-        it 'returns "/feed"' do
-          expect(blog.feed_path).to eql "/feed"
+        context ":feed_path" do
+          it 'returns "/feed"' do
+            expect(blog.feed_path).to eql "/feed"
```

Comment:
It feels like this should be blog.feed_path rather than the actual value. If the factory changes, then the value will change and the test will fail. That would keep changes in one place only without consequence.

Reply:
I changed the building.label test, but I think the blog tests should stay the way they are.  We are testing 2 different contexts and expecting different text to be returned for each one.  I think using the expected text makes it easier to understand what the expectations are.